api/src/main/java/org/openmrs/api/context/Context.java

```diff
@@ -267,6 +267,9 @@ public static void setContext(ServiceContext ctx) {
 	}
 
 	/**
+	 * @deprecated since 2.3.0
```

Comment:
Can you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Deprecation

Reply:
Combining deprecation with since can be confusing to some users because of the @since annotation. So i would recommend doing exactly as stated at https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Deprecation